lib/bitreserve/options.rb

```diff
@@ -0,0 +1,33 @@
+module Bitreserve
+  module Options
+    attr_reader :options
+    API_BASE = 'https://api.bitreserve.org'
```

Comment:
I was suposed to have deleted this, but then checked out the file for other reasons and this came back